src/Tribe/Cache.php

```diff
@@ -58,15 +57,27 @@ public function set_transient( $id, $value, $expiration = 0, $expiration_trigger
 	}
 
 	/**
-	 * @param string $id
-	 * @param string $expiration_trigger
+	 * Get cached data. Optionally set data if not previously set.
+	 *
+	 * @param string       $id                 The key for the cached value.
+	 * @param string       $expiration_trigger Optional. Hook to trigger cache invalidation.
+	 * @param string|array $callback_if_empty  Optional. Sets the cache value to the return of this callback if no value was already set.
+	 * @param mixed        $callback_args      Optional. Args passed to callback.
+	 * @param int          $expiration         Optional. Value of expiration if we have to set the cache value.
 	 *
 	 * @return mixed
 	 */
-	public function get( $id, $expiration_trigger = '' ) {
+	public function get( $id, $expiration_trigger = '', $callback_if_empty = null, $callback_args = array(), $expiration = 0 ) {
```

Comment:
Bonus points if you add a test for this to
* test that the `default` parameter will take and use scalar values
* test that the `default` parameter will take and use callbacks (called only once)
* test that the `args` array, if present, is passed to the `default` callback
* test that if the `default` callback throws an exception we handle the exception and do not fill the cache